tests/system/aiplatform/test_model_monitoring.py

```diff
@@ -156,22 +176,66 @@ def test_mdm_two_models_one_valid_config(self):
             gca_obj_config.prediction_drift_detection_config == drift_config.as_proto()
         )
 
+        # delete this job and re-configure it to only enable drift detection for faster testing
+        job.delete()
         job_resource = job._gca_resource.name
 
-        # test job update and delete()
-        timeout = time.time() + 3600
-        new_obj_config = model_monitoring.ObjectiveConfig(skew_config)
+        # test job delete
+        with pytest.raises(core_exceptions.NotFound):
```

Comment:
Preference for each additional test to be added as a new test method. The test methods get executed in order. Reference example: https://github.com/googleapis/python-aiplatform/blob/5fe5ae5ccc58a547060cbe04539be963930d5828/tests/system/aiplatform/test_experiments.py#L44

Reply:
I have separated out the test cases